Tactility/Source/PartitionsEsp.cpp

```diff
@@ -20,7 +20,8 @@ static esp_err_t initNvsFlashSafely() {
     return result;
 }
 
-static wl_handle_t data_wl_handle = WL_INVALID_HANDLE;
+// Removed static so it's global
+wl_handle_t data_wl_handle = WL_INVALID_HANDLE;
 
 size_t getSectorSize() {
 #if defined(CONFIG_FATFS_SECTOR_512)
```

Comment:
I'd rather not use global variables.
Can you make a function like `getDataPartitionWlHandle()` and expose it via `Tactility/Private/PartitionsEsp.h`?
The variable can then become static again.